source/desktop/fastdesk/signout.md

```diff
@@ -36,9 +36,11 @@ As illustrated in **image 2**, clicking on the icon (outlined in orange) will br
 
 **_This instruction guide should assist you in learning how to effectively sign out of your FastDesk session. If you have any questions or still require assistance, please contact the FastDesk support team on 0800 923 0617_**.
```

Comment:
swap this to a note format instead, and change the 'Please be aware' note above that to a warning, so that a) you don't have two notes next to one another and b) that people know how to sign out right, why not